utility/pbkdf2-hash-generator/README.md

```diff
@@ -0,0 +1,10 @@
+# pbkdf2-hash-generator
+This contains the Client for generate PBKDF2 Hash and the Salt Value for a given plain text password.
+
+## How to use this Hash Generator
+
+1. Clone this repo.
+2. Build this component using `mvn clean install`
+3. Then run the main method of this java project.
```

Comment:
let's provide steps to run from the build jar other than the java project.

Reply:
Addressed with https://github.com/wso2-extensions/identity-hash-provider-pbkdf2/pull/11/commits/49539a8dafdea93ba8b6d384fa5554269fdbbe05